src/fenicsxconcrete/sensor.py

```diff
@@ -0,0 +1,67 @@
+
+from pydantic.dataclasses import dataclass
+from typing import Callable
+import numpy as np
+
+
+
+
+@dataclass(config=dict(arbitrary_types_allowed=True))
+class PointSensorDefinition:
+    """
+    Definition of a point sensor
+
+    Args:
+        location: The location of the sensor
+        variable: The variable to be measured
+     
+    """
+    location: tuple[float,float,float]
+    variable: str
```

Comment:
I don't know yet. The enum would have the benefit that the input can very easily be checked, but it would also be restrictive. If you have a new material model with weird history variables, it would be easier to use the sting